sphinx/transforms/__init__.py

```diff
@@ -294,23 +295,40 @@ class UnreferencedFootnotesDetector(SphinxTransform):
     Detect unreferenced footnotes and emit warnings
     """
 
-    default_priority = 200
+    default_priority = Footnotes.default_priority + 1
```

Comment:
Could this change of priority be breaking extensions that would have relied on the absolute priority? 

I'm worried that some extensions could have fixed their internal footnotes by adding a priority `UnreferencedFootnotesDetector.default_priority - 1` which would now hit the same priority as the footnotes transformation.

Reply:
Well, I think there would be a very slim chance of that 😅, and obviously this transform doesn't actually make any changes to the doctree so it won't have any indirect effect on other transforms.

If you really want, I can change the existing transform to a "no-op", then add this as an additional new transform?

It's of note, that I'm doing this as a result of https://github.com/executablebooks/MyST-Parser/pull/931; so that I can remove ad-hoc fixes in myst-parser and just let sphinx handle warnings

Reply:
No let's not overcomplicate the things. But add a CHANGELOG entry for that by saying that the priority has been changed (I don't know if we have it, but a table with the transformations and post-transformations priorities would be nice...).

Reply:
> I don't know if we have it, but a table with the transformations and post-transformations priorities would be nice...

Nope I don't believe so, it's on the TODO list 😄